src/exchange.ts

```diff
@@ -164,10 +161,11 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
       keys.push(op.key);
 
       if (!ops.has(op.key)) {
-        const isNetworkOnly = op.context.requestPolicy === 'network-only';
         ops.set(
           op.key,
-          isNetworkOnly ? toRequestPolicy(op, 'cache-and-network') : op
+          op.context.requestPolicy === 'network-only'
```

Comment:
Oh I suppose we could use getRequestPolicy here

Reply:
-1 byte for the win 👍